README.md

```diff
@@ -33,6 +33,30 @@ To check the layout change, in your HTML file, add attribute in your html tags
 
 Or, in your js file, set `document.dir = 'rtl'` or `document.dir = 'ltr'`.
 
+## Options
+
+postcss-bidirection accepts an options object.
+
+```
+const plugin = require('postcss-bidirection');
+const opts = {
+    ...
+};
+postcss([ plugin(opts) ]).process(input) ...
+```
+
+### Custom Selectors
+
+By default, postcss-bidirection will prefix generated selectors with `html[dir="rtl"]` or `html[dir="ltr"]`.
+The `buildSelector` option allows you to customize this behavior. For example, to drop `html` from generated selectors, pass a custom `buildSelector` function to the plugin:
+
+```
+const opts = {
+  buildSelector = function(selector, direction) {
+    return '[dir=" + direction + '"] ' + selector;
+  }
+};
+```
```

Comment:
please document the output as well, so it will be more easy to know what is changed

Reply:
Thank you!

I'm not sure that I understand what you mean by "document the output". I modified the readme (in Options -> Custom Selectors) and it contains an example. Do you have something else in mind?  Do you want me to comment that example?
By the way I'm thinking maybe this section should be rather moved at the bottom of the readme.